Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_BezierPointer.cs

```diff
@@ -37,6 +40,13 @@ public class VRTK_BezierPointer : VRTK_WorldPointer
         private GameObject curvedBeamContainer;
         private CurveGenerator curvedBeam;
 
+        // materials of customPointerCursor and teleportBeam (if defined)
+        private Material customPointerMaterial;
+        private Material beamTraceMaterial;
+        [Tooltip("Adapt tracer instances to the curve length")]
+        [SerializeField]
+        private bool rescalePointerTracer = false;
```

Comment:
Why is this a private field if you want it to appear in the inspector?

Wouldn't this just prevent it being changed via code?